capellambse/extensions/validation/rules.py

```diff
@@ -0,0 +1,430 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capellambse contributors
+# SPDX-License-Identifier: Apache-2.0
+from __future__ import annotations
+
+import re
+import typing as t
+
+from capellambse.model import common as c
+from capellambse.model.crosslayer import fa
+from capellambse.model.layers import ctx as sa
+
+from . import _validate
+
+
+@_validate.register_rule(
+    category=_validate.Category.REQUIRED,
+    types=[sa.SystemComponent, sa.SystemFunction, sa.Capability],
+    id="Rule-001",
+    name="Object has a description or summary",
+    rationale=(
+        "A comprehensive description or summary for an object is "
```

Comment:
Having such verbose truisms in teh rationales,  which repeatedly use general terms as efficiency etc. would unnecessarily bloat every report....

Reply:
agree, lets reduce the bloat where we can. I'll make some changes here and there.